internal/handlers/pg/pgdb/query.go

```diff
@@ -110,15 +110,15 @@ func (pgPool *Pool) QueryDocuments(ctx context.Context, tx pgx.Tx, sp *SQLParam)
 }
 
 // GetDocuments returns an queryIterator to fetch documents for given SQLParams.
-// If the collection doesn't exist, it returns nil and no error.
+// If the collection doesn't exist, it returns an empty iterator and no error.
 // If an error occurs, it returns nil and that error, possibly wrapped.
 func (pgPool *Pool) GetDocuments(ctx context.Context, tx pgx.Tx, sp *SQLParam) (
 	iterator.Interface[uint32, *types.Document], error,
 ) {
 	q, args, err := buildQuery(ctx, tx, sp)
 	if err != nil {
 		if errors.Is(err, ErrTableNotExist) {
-			return nil, nil
+			return newIterator(ctx, nil), nil
```

Comment:
@AlekSi here https://github.com/FerretDB/FerretDB/pull/1487/files#r1032112552 you proposed to return an empty iterator if the queried collection doesn't exist. Does my implementation in this PR cover your idea, or did you mean something else?

Reply:
I think so, but it will be more clear with Close method added

Reply:
@AlekSi you probably missed that, but `Close` was added and merged :)

Reply:
Got confused by GH collapsing a diff at the worst possible place